src/pwd/pwd.c

```diff
@@ -1,6 +1,95 @@
+#include <limits.h>
+#include <stdbool.h>
 #include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
 
-int main(void) {
-  printf("Hello, World!\n");
-  return 0;
-}
\ No newline at end of file
+#define NAME "pwd (canoutils)"
+#define VERSION "1.0.0"
+#define AUTHOR "Mehadav Itay (whiteapolo)"
+
+#define print_version()                                                        \
+  do {                                                                         \
+    printf("%s\nversion: %s\nby: %s\n", NAME, VERSION, AUTHOR);                \
+  } while (0)
+
+char *realpath(const char *path, char *resolved_path);
```

Comment:
`realpath` is not part of the C99 standard, it is defined to `4.4BSD, POSIX.1-2001`.